src/components/table/table.stories.mdx

```diff
@@ -0,0 +1,121 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import { withKnobs, boolean, select } from '@storybook/addon-knobs';
+import basicDemo from './demo/basic.twig';
+import data from './demo/data.json';
+import './table.scss';
+
+<Meta title="Components/Table" decorators={[withKnobs]} />
+
+# Table Object
+
+The `c-table` class applies default styling to HTML `table` elements.
+A table can optionally headers and footers.
+
+<Preview>
+  <Story name="Basic">
+    {basicDemo({
+      hasHeader: boolean('Include Header', true),
+      hasFooter: boolean('Include Footer', true),
+      rowModifier: select('Table Row Modifier Class', [
+        '',
+        'c-table--ruled',
+        'c-table--striped'
+      ]),
```

Comment:
As written, this makes it seem like these modifiers can't be combined. I don't believe that's the case: Should these be their own distinct options instead of a `<select>` element?

Reply:
Good call. I was thinking that might feel heavey-handed but I tried it out and it looked okay. I'll update this to allow both.

Reply:
Done!